src/services/ReviewSummationService.js

```diff
@@ -307,10 +327,15 @@ patchReviewSummation.schema = {
   entity: joi.object().keys({
     scoreCardId: joi.id(),
     submissionId: joi.string().uuid(),
-    aggregateScore: joi.score(),
+    aggregateScore: joi.score().when('status', {
+      is: joi.string().valid('queued', 'processing').required(),
```

Comment:
@callmekatootie we should move these `enum` values to `config/default.js` file; so that in future if we want to add new status; don't have to update the code

Reply:
applicable for `completed` status as well

Reply:
@meshde Do you want to take this up?

Reply:
Sure, will do 👍🏻